openapi/openapi.yaml

```diff
@@ -0,0 +1,299 @@
+openapi: 3.1.0
+info:
+  title: USDR standard representation for federal grant data
+  version: 1.0.0
+paths: {} # No endpoints defined
+components:
+  schemas:
+      AdditionalInformation:
+        type: object
+        properties:
+          eligibility:
+            type: string
+          text:
+            type: string
+          url:
+            type: string
+            format: uri
+      Agency:
+        type: object
+        properties:
+          code:
+            type: string
+          name:
+            type: string
+      Applicant:
+        type: object
+        properties:
+          name:
+            type: string
+            enum:
+              - 'State governments'
+              - 'County governments'
+              - 'City or township governments'
+              - 'Special district governments'
+              - 'Independent school districts'
+              - 'Public and State controlled institutions of higher education'
+              - 'Native American tribal governments (Federally recognized)'
+              - 'Public housing authorities/Indian housing authorities'
+              - 'Native American tribal organizations (other than Federally recognized tribal governments)'
+              - 'Nonprofits having a 501(c)(3) status with the IRS, other than institutions of higher education'
+              - 'Nonprofits that do not have a 501(c)(3) status with the IRS, other than institutions of higher education'
+              - 'Private institutions of higher education'
+              - 'Individuals'
+              - 'For profit organizations other than small businesses'
+              - 'Small businesses'
+              - 'Others (see text field entitled "Additional Information on Eligibility" for clarification)'
+              - 'Unrestricted (i.e., open to any type of entity above), subject to any clarification in text field entitled "Additional Information on Eligibility'
+          code:
+            type: string
+            enum:
+              - '00'
+              - '01'
+              - '02'
+              - '04'
+              - '05'
+              - '06'
+              - '07'
+              - '08'
+              - '11'
+              - '12'
+              - '13'
+              - '20'
+              - '21'
+              - '22'
+              - '23'
+              - '25'
+              - '99'
+      Award:
+        type: object
+        properties:
+          ceiling:
+            type: string
+          floor:
+            type: string
+          estimated_total_program_funding:
+            type: string
+          expected_number_of_awards:
+            type: string
+        required:
+          - ceiling
+          - floor
+      CloseDate:
+        type: object
+        properties:
+          date:
+            type: string
+            format: date
+          explanation:
+            type: string
+      Email:
+        type: object
+        properties:
+          email:
```

Comment:
recommend changing to `address` will be better to do `Email.address` rather than `email.email`